utils/phpstan-extensions/src/Rule/PathsAreNotTooLongRule.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PHPStanExtensions\Rule;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PhpParser\Node\Expr\ClassConstFetch;
+use Rector\Core\PhpParser\Node\CustomNode\FileNode;
```

Comment:
Should this be `use PHPStan\Node\FileNode;` instead?

Reply:
Yes, it's s PHPStan rule